cli/cmd/vm_ssh_endpoint.go

```diff
@@ -53,3 +68,34 @@ Visit https://vendor.replicated.com/account-settings to link your account`)
 
 	return nil
 }
+
+func (r *runners) getVMIDFromArg(arg string) (string, error) {
+	if _, err := r.kotsAPI.GetVM(arg); err == nil {
+		return arg, nil
+	}
+
```

Comment:
getVMIDFromArg function will mask valid errors from the user.  If GetVM returns ErrNotFound or ErrForbidden, then we can call ListVMs. Otherwise this function should fail. ErrForbidden can be returned due to RBAC rules.